spacetemplate/assets/scrum.yaml

```diff
@@ -0,0 +1,242 @@
+space_template:
+  id: "629c963a-174c-4c37-b487-272067e88bd4"
+  name: Scrum
+  description: >
+    Scrum is an agile framework for managing work with 
+    an emphasis on software development.
+work_item_types:
+- id: &commonWorkItemTypeID "ee0fc96a-87b3-475a-9937-ae6aa22d9eb9"
+  extends: "86af5178-9b41-469b-9096-57e5155c3f31"
+  name: Common Work Item Type
+  description: TBD
+  fields:
+    "system.history": 
+      label: History
+      description: The history of changes of the workitem
+      type:
+        kind: history
+      required: no
+    "system.reason": 
+      label: Reason
+      description: TBD
+      type:
+        kind: string
+      required: no
+    "system.stackrank": 
+      label: StackRank
+      description: TBD
+      type:
+        kind: float
+      required: yes
+- id: &impedimentID "35837274-b833-4001-bee9-4d69e7f8aa0c"
+  extends: "ee0fc96a-87b3-475a-9937-ae6aa22d9eb9"
```

Comment:
@baijum @DhritiShikhar please use `extends: *commonWorkItemTypeID` here instead to make it more obvious from where you extend.